lua/terrortown/entities/roles/sidekick/shared.lua

```diff
@@ -238,14 +249,14 @@ if SERVER then
 		end
 
 		if sikis then
-			local enabled = GetConVar("ttt2_siki_mode"):GetBool()
+			local enabled = GetConVar("ttt2_siki_mode"):GetInt() == 1
 
 			for _, siki in ipairs(sikis) do
 				if not IsValid(siki) or not siki:IsPlayer() or not siki:IsActive() then continue end
 
 				siki:SetNWEntity("binded_sidekick", nil)
 
-				if not enabled then continue end
+				if enabled then continue end

```

Comment:
the condition should be negated again

Reply:
my bad, ill fix it when get home

Reply:
done with bc3810fc5f1431be2a02d1c628218ba1e39579af